src/main/proto/ga4gh/pipeline_metadata.proto

```diff
@@ -1,8 +1,7 @@
 // This file defines the metadata objects.
 syntax = "proto3";
 
-import "google/protobuf/struct.proto";
```

Comment:
please double check this

Reply:
Is struct.proto part of the source protobuf? I can't seem to find it or anywhere it's used in schemas

Reply:
No, it isn't. It however does seem to be importable. @zbozoky  what do you think? My guess is it's the `built-in` modules from the protobuf.

Reply:
It is a built-in module. We have to be careful with the package name, because there is a **google**/api under the ga4gh package and a **google**/protobuf/struct... under the google package. However, if we do not use struct.proto in the file, we should just remove it. I do not understand why it was imported in the first place.